docs/Methods.md

```diff
@@ -0,0 +1,390 @@
+# Methods list & usage
+
+## 1. Admin method
+
+### 0-1. Install swap contract
+
+```bash
+clif contract run wasm swap_install.wasm 0.1 --from elsa
+```
+
+### 0-2. Check contract address and memorize
+
+```bash
+clif contract query $(clif keys show elsa -a) ""
```

Comment:
```$ clif contract query **address** $(clif keys show elsa -a)```

Need to address & do not need to `""`

Reply:
https://github.com/hdac-io/swap/pull/1/commits/e4c59e4a20ee83c736e2e3823399e02540ef2d55